src/Driver/OCI8/Result.php

```diff
@@ -136,4 +158,47 @@ private function fetchAll(int $mode, int $fetchStructure): array
 
         return $result;
     }
+
+    /**
+     * Screen PHP error messages for any error that the OCI8 extension
+     * doesn't report via oci_error()
+     *
+     * @link http://www.dba-oracle.com/t_error_code_list.htm
+     */
+    public static function handleError(
```

Comment:
Please move the implementation under the callback set by `set_error_handler()`. There's no need to keep a dedicated public function.